sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -256,10 +303,9 @@
      * of how the partitions are distributed among the splits.
      */
     public Read<K, V> withTopics(List<String> topics) {
```

Comment:
This seems just as easy to do in a separate PR, since it's unrelated to the current change.